.github/workflows/build.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           node-version: "16"
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
-      - run: cd ethereum && make test
+      - run: cd ethereum && make test && make test-push0
```

Comment:
maybe this could have just been put under `test` as well?

Reply:
Yeah this was deliberate actually to avoid coupling these tests together. In the future when we do want to support PUSH0 instructions in our deployments we can just edit this line back to what it was, but still keep the command around in the Makefile available for use